modules/boot/include/commands.h

```diff
@@ -14,7 +14,7 @@
 #define TIMER_RESET_BUTTONS (CButton::Z | CButton::B)
 #define FREE_CAM_BUTTONS (CButton::Z | CButton::B | CButton::A)
 #define MOVE_LINK_BUTTONS (CButton::L | CButton::R | CButton::Y)
-#define FRAME_PAUSE_BUTTONS (CButton::R | CButton::DPAD_UP)
+#define FRAME_PAUSE_BUTTONS (CButton::R | CButton::DPAD_LEFT)
```

Comment:
was this a request somebody had?

Reply:
nah i just thought it was annoying that it was the same combo as teleport